src/porepy/grids/match_grids.py

```diff
@@ -608,7 +608,7 @@ def structured_refinement(
     if g.dim == 1:
         # Rotate coarse nodes and fine cell centers to align with the x-axis
         tangent = pp.map_geometry.compute_tangent(nodes)
-        reference = [1, 0, 0]
+        reference = np.array([1.0, 0, 0])
```

Comment:
Probably not needed? But consistent with other change in this PR, I think.
```suggestion
        reference = np.array([1.0, 0.0, 0.0])
```